tests/parallel_test/cases.task

```diff
@@ -236,6 +236,7 @@
 30,,script,./test.sh -f general/import/commit.sim 
 30,,script,./test.sh -f general/compute/diff2.sim 
 30,,develop-test,bash 3-connectors/R/test.sh
+30,,develop-test,bash 3-connectors/c#/test.sh
```

Comment:
@gccgdb1234  not sure if the C# connector examples need to be added in regular CI if the test consumes a few seconds or longer. And if other connectors' need to be added too if C# should